app/src/main/java/com/asfoundation/wallet/App.kt

```diff
@@ -253,4 +257,6 @@ class App : MultiDexApplication(), BillingDependenciesProvider {
 
   override fun ewtObtainer() = ewtObtainer
 
+  override fun oemIdExtractorService() = oemIdExtractorService
```

Comment:
this injection and access through the view might not be needed here. Just injecting directly the "oemIdExtractorService: OemIdExtractorService" in the constructors didn't work?